go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/kanopy-platform/cdnvalidator
 go 1.16
 
 require (
+	github.com/aws/aws-sdk-go v1.43.4
```

Comment:
We probably want to use the [aws-sdk-gov2 ](https://github.com/aws/aws-sdk-go-v2) since it was GAed.